src/openms/source/SYSTEM/File.cpp

```diff
@@ -697,7 +697,7 @@ namespace OpenMS
     {
       for (const String& fn : exe_filenames)
       {
-        if (exists(p + fn))
+        if (exists(p + fn) && !isDirectory(p + fn))
```

Comment:
line https://github.com/OpenMS/OpenMS/pull/4397/files#diff-d39cf05fce87e1511bd1ca97fcee03d6R682
should also get this check.

Reply:
True! thanks